### ISSUE TITLE
FEATURE REQUEST : Bigger coordinate system on visualizer.

### ISSUE THREAD

**Issue by macrumpel** [Sienci-Labs/gsender#584]
Hello,

on my screen the numbers of the coordinate system are very small and even impossible to read. It would be great to have this adjustable, as precision is important.
Thanks a lot !

Hubertus

**Comment by kglovern** [comment]
Do you mean on the visualizer itself?

**Comment by macrumpel** [comment]
Yes exactly. Just to verify that the object is in accordance to placement on the board. I use gSender for plotting, and it would help me to be sure that the drawing is on the paper.

**Comment by macrumpel** [comment]
What would also be great is the ability to have the axis visible even if you zoom in. More like rulers actualle...

**Comment by kglovern** [comment]
A large number of visualizer customizations including larger numbers, machine bed matching, and more are coming to 1.7.0-Edge so I'll be closing this as complete.
